OpenInSublimeText/OSTOpenInSublimeText.m

```diff
@@ -112,7 +112,7 @@ - (void) openCurrentFileWithSublimeTextAtLocation:(CursorLocation)location {
         
         NSTask *task = [[NSTask alloc] init];
         
-        task.launchPath = @"/Applications/Sublime Text.app/Contents/SharedSupport/bin/subl";
+        task.launchPath = @"/Applications/Sublime Text 2.app/Contents/MacOS/Sublime Text 2";
```

Comment:
If it's possible, it'd be ideal to support Sublime Text 2 and 3. Sublime Text 3 is still in Beta.